build/UserALEWebExtension/optionsPage.html

```diff
@@ -19,31 +19,30 @@
 <html>
 <head>
     <title>User ALE Web Extension - Options</title>
-    <script src="globals.js"></script>
     <script src="options.js"></script>
     <meta charset="utf-8">
 </head>
 <body>
 <h1>Options</h1>
 <form>
     <label>User ALE Server Host:</label>
-    <input id="host"/>
+    <input id="url"/>
     <br/>
     
     <label>User ALE Client Script:</label>
     <input id="clientScript"/>
     <br/>
 
     <label>User:</label>
-    <input id="toolUser"/>
+    <input id="user"/>
     <br/>
 
     <label>Tool Name:</label>
-    <input id="toolName"/>
+    <input id="tool"/>
```

Comment:
Is there a reason this differs from the javascript package naming conventions?

Reply:
I changed the html id's to match the data tag names defined here:
https://github.com/apache/flagon-useralejs/blob/aac8e6766972e27bb593e777f0c9ca5ec84598ab/src/getInitialSettings.js#L25

Originally I intended to add a layer of abstraction that defines the tag names and js property names for each setting and could be imported into getInitialSettings.js and options.js. But I wanted to limit the scope of this PR to the web extension.